tests/cases/fourslash/callHierarchyFunction.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path="fourslash.ts" />
+
+//// [|function [|foo|]() {
+////     [|bar|]();
+//// }|]
+////
+//// [|function /**/[|bar|]() {
+////     [|baz|]();
+////     [|quxx|]();
+////     [|baz|]();
+//// }|]
+////
+//// [|function [|baz|]() {
+//// }|]
+////
+//// [|function [|quxx|]() {
+//// }|]
+
+const [
+    fooFunctionRange,
+    fooFunctionSelectionRange,
+    barReferenceRange,
+    barFunctionRange,
+    barFunctionSelectionRange,
+    bazReferenceRange1,
+    quxxReferenceRange,
+    bazReferenceRange2,
+    bazFunctionRange,
+    bazFunctionSelectionRange,
+    quxxFunctionRange,
+    quxxFunctionSelectionRange,
+] = test.ranges();
+
+goTo.marker();
+verify.callHierarchy({

```

Comment:
So, hear me out, might it be nicer to baseline a graphical representation of the call heirarchy results, rather than all this explicit, manual shaping?

Something like `verify.callHierarchy()` which dumps
```
[r1|function [r2|foo|]() {
    [r3|bar|]();
}|]

[r4|function /**/[r5|bar|]() {
    [r6|baz|]();
    [r7|quxx|]();
    [r8|baz|]();
}|]

[r9|function [r10|baz|]() {
}|]

[r11|function [r12|quxx|]() {
}|]

┏foo
┃ name: r2
┃ selection: r1
┃ references: r3
┗┳ bar (selected)
 ┃ name: r5
 ┃ selection: r4
 ┗┳ baz
  ┃ name: r10
  ┃ selection: r9
  ┃ references: r6, r8
  ┗ quxx
    name: r12
    selection: r11
    references: r7
 
```
or something else like what the editor would show for the query into a baseline file?

Reply:
I've switched the tests to baselines. Let me know what you think of the output format. Since I'm not using them anymore, should I just go ahead and get rid of the old `verifyCallHierarchy...` methods?

Reply:
Yeah, I guess so. I like the baseline format, but it could do with being a bit more... compact/diffable. I liked what I suggested above because the query result is pretty close to what the editor displays for the query (albeit with text instead of an icon for the `kind`), plus a bit of markup to link it back to the generated spans.

Reply:
The arrows don't show well in beyond compare, unfortunately. I also think the way I'm highlighting spans is easier to read than `[r1|function|]` etc.